input/data/manure/refreshed_manure_management.json

```
@@ -35,5 +35,25 @@
             "potassium_efficiency": 0.23,
             "ash_efficiency": 0.20
           }
+    ],
+    "storage": [
+      {
+        "name": "slurry_storage_outdoor_1",
+        "type": "slurry storage outdoor",
+        "capacity": 100,
```

I think we can also make this value null for now, or even omit it from the default manure input? I believe it is not utilized yet and it could be misleading for users to think does make a difference in calculations
```suggestion
        "capacity": null,
```